### DISCUSSION TITLE
vits2 training from the beginning with my own dataset

### DISCUSSION THREAD

**Discussion by wjdwn819** [p0p4k/vits2_pytorch#102] [Q&A]
Hi, I have been training a Korean TTS model with vits and now I would like to start training with VITS2.
Would it be possible to start training from the 0.pth (without a pretrained model) with the inference you made in vits2_pytorch?
Or does it require a pretrained model with your "vits2_pytorch"
I really appreciate your hard work. Thank you.

**Comment by p0p4k** [comment]
You can do from scratch without using pretrained model.
